controllers/login.controller.js

```diff
@@ -0,0 +1,40 @@
+const db = require("../database/connection");
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+module.exports = async (req, res) => {
+  const { username, password } = req.body;
+  try {
+    const results = await db.query("SELECT * FROM users WHERE username = $1", [
+      username,
+    ]);
+    console.log(results.rows);
+    //
+    if (results.rows.length === 0) {
+      return res.status(403).send({
+        success: false,
+        message: "username does not exist",

```

Comment:
[from @saherhanif pull request](https://github.com/WebAhead-cs2/caps-look-api/pull/16/files#diff-f037f7d0a205fe6420cabc55940e9c9e79c5477006e4fdffb552ac4f3b5005a9R95) there is no username, instead we're using `email`. so just a small change in the query line7 and message line15 and line26